examples/haas.cfg.dev-no-hardware

```diff
@@ -0,0 +1,96 @@
+# This is an example configuration file for HaaS. The ``haas`` command will
+# read its  configuration from a file called ``haas.cfg``, in the current
+# working directory. The ``haas.wsgi`` script will look for it at
+# ``/etc/haas.cfg``. This will change in the future; it isn't considered
+# reasonable behavior.
+#
+# The comments in this file are targeted at developers; Take a look at
+# haas.cfg.example for something more production/end-user oriented.
+#
+# The options under [client] are required for making client-side requests with
+# the cli tool (anything but ``haas serve`` or ``haas serve_networks``). Driver
+# specific options are mandatory if and only if that driver is in use.  The
+# database section is needed to run the server at all, while the remaining
+# options can be omitted if the dry_run option is enabled.
```

Comment:
Perhaps add some text explaining how this file differs from `examples/haas.cfg.dev-example`, and a similar cross-reference in that file.